docker-pactbroker-compose.yml

```diff
@@ -1,9 +1,26 @@
 version: "3"
 
 services:
+  postgres:
```

Comment:
do you use postgres in ia-home-office-integration?

Reply:
No, I will remove it

Reply:
Removed the pact broker compose yaml